packages/curve-ui-kit/src/features/candle-chart/ChartWrapper.tsx

```diff
@@ -1,14 +1,14 @@
 import { useMemo, useRef, useState } from 'react'
 import { styled } from 'styled-components'
-import { useTheme } from '@mui/material/styles'
 import Button from 'ui/src/Button/Button'
-import CandleChart from 'ui/src/Chart/CandleChart'
-import DialogSelectChart from 'ui/src/Chart/DialogSelectChart'
-import DialogSelectTimeOption from 'ui/src/Chart/DialogSelectTimeOption'
-import { useChartPalette } from 'ui/src/Chart/hooks/useChartPalette'
 import Checkbox from 'ui/src/Checkbox'
 import Icon from 'ui/src/Icon'
 import Spinner, { SpinnerWrapper } from 'ui/src/Spinner'
+import { useTheme } from '@mui/material/styles'
+import CandleChart from '@ui-kit/features/candle-chart/CandleChart'
```

Comment:
Personally when I'm importing things that are specific to the feature or component itself, I use relative imports. Found that easier when stuff might move again in the future